src/domains/ecommerce-app/04_apim_ecommerce_checkout.tf

```diff
@@ -100,7 +100,7 @@ resource "azurerm_api_management_api_operation_policy" "delete_transaction" {
   xml_content = file("./api/ecommerce-checkout/v1/_validate_transactions_jwt_token.tpl")
 }
 
-resource "azurerm_api_management_api_operation_policy" "get_payment_request_info_api" {
+resource "azurerm_api_management_api_operation_policy" "get_payment_request_info_api_policy" {
```

Comment:
to update xml content with _payment_request_policy.xml.tpl

Reply:
where's my head? :D